src/ui/components/shared/UserSettingsModal/panels/Experimental.tsx

```diff
@@ -2,22 +2,7 @@ import { config } from "shared/user-data/GraphQL/config";
 import { PreferencesKey } from "shared/user-data/GraphQL/types";
 import { BooleanPreference } from "ui/components/shared/UserSettingsModal/components/BooleanPreference";
 
-export const PREFERENCES: PreferencesKey[] = [
-  "feature_columnBreakpoints",
-  "backend_profileWorkerThreads",
-  "feature_basicProcessingLoadingBar",
-  "console_showFiltersByDefault",
-  "backend_disableScanDataCache",
-  "feature_brokenSourcemapWorkaround",
-  "backend_enableRoutines",
-  "backend_rerunRoutines",
-  "backend_disableRecordingAssetsInDatabase",
-  "feature_reactPanel",
-  "feature_reduxDevTools",
-  "backend_disableIncrementalSnapshots",
-  "backend_disableConcurrentControllerLoading",
-  "backend_disableProtocolQueryCache",
-];
```

Comment:
The following keys were removed from `Experimental` and were not re-added to `Advanced`
* `console_showFiltersByDefault` -> Moved to preferences
* `feature_reactPanel` -> Disappeared
* `feature_reduxDevTools` -> Disappeared

I _assume_ this was intentional but it wasn't explicitly mentioned in the description so I want to verify.

Reply:
`console_showFiltersByDefault` moved to preferences. The other two stayed in experimental

Reply:
Gotcha. I must have overlooked it below.